mutacc_auto/parse/parse_scout.py

```diff
@@ -39,3 +40,98 @@ def get_cases_from_scout(scout_output, days_ago=None):
             recent_cases.append(case)
 
     return recent_cases
+
+
+
+
+
+def get_vcf_from_json(scout_vcf_output):
+
+    """
+        Reconstructs vcf from scout variant object
+
+        Args:
+            scout_vcf_output (str): string returned by command 'scout export variants --json'
+
+        Returns:
+            vcf_string (str): string with vcf content
+    """
+
+    scout_vcf_output = json.loads(scout_vcf_output)
+
+    vcf_string = ""
+
+    #Write header of vcf
+    for header_line in HEADER:
+        vcf_string += header_line + '\n'
+
+    #Get samples
+    samples = [sample['sample_id'] for sample in scout_vcf_output[0]['samples']]
+    samples = '\t'.join(samples)
+
+    vcf_string += f"#CHROM\tPOS\tID\tREF\tALT\tQUAL\tFILTER\tINFO\tFORMAT\t{samples}\n"
```

Comment:
I would like a simple "TAB.join(#CHROM, POS, ...etc)" if possible or even better
header_column_names = ["#CHROM", "POS", ..etc]
TAB.join(header_column_names, samples) + NEWLINE

That is a better description of what the parts are and more readable. You will have to excuse my python, but I think you get the idea